### ISSUE TITLE
Doesn't detect RS485 to USB com port

### ISSUE THREAD

**Issue by nmambre** [julihirn/Serial-Monitor#3]
I did a quick test using com0com with 2 virtual ports and it worked, but a real port COM5, doesn't show up in the list. Is it possible to write the port number instead of selecting? I've opened and closed the program various times to see if it recognizes COM5. 

![devicemanager](https://github.com/julihirn/Serial-Monitor/assets/15265357/760e5586-113a-4a78-a7c8-61958f27d626)
![driver](https://github.com/julihirn/Serial-Monitor/assets/15265357/8e7b5cfc-c1cb-4fc3-9b74-337d7a4a4c35)
![sercomms](https://github.com/julihirn/Serial-Monitor/assets/15265357/d3e4bbd3-33d4-4ebe-90c8-5f3d6e17898d)

**Comment by julihirn** [comment]
Yes, you can manually assign a port via: Channel>Properties. 
![image](https://github.com/julihirn/Serial-Monitor/assets/94691568/6cc34863-887f-471c-9b5d-5c1c8f753f6e)

As for the bug, thanks for letting me know. I'll have a look at this now, give me about 30 minutes.

**Comment by julihirn** [comment]
I've noticed I have the same RS485 device and noticed that the device is indeed not showing. As a temporary measure, I have updated the binaries to include a setting to change how devices are listed. 

The new setting can be accessed via: _Tools>Options_ (CTRL+I), and check this setting: _Use Legacy Port Listing_. The device should now appear when you go to select a port.

**Latest Version:**
https://github.com/julihirn/Serial-Monitor/releases/tag/serial-monitor-1.5.2
